commands/article.js

```diff
@@ -0,0 +1,28 @@
+const config = require('../config.json');
+const Parser = require('rss-parser');
+const parser = new Parser();
+const random = require('random');
+
+
+
+module.exports.execute = async (client, message, args) => {
+	const feed = await parser.parseURL('https://knightsofacademia.org/category/articles/feed');
+	const r = random.int(0 , feed.items.length);
+	let response = `Here is a article i found called "${(feed.items[r].title)}" here is the link to it ${feed.items[r].link}`;
```

Comment:
It's not necessarily clear to the user that this article was randomly selected. How about something like:
`Here's a random article from KOA titled "${(feed.items[r].title)}": ${feed.items[r].link}`